website/docs/reference/resource-properties/deprecation_date.md

```diff
@@ -0,0 +1,32 @@
+---
+resource_types: [models]
+datatype: deprecation_date
+required: no
+---
+
+<File name='models/<schema>.yml'>
+
+```yml
+models:
+  - name: my_model
+    description: deprecated
+    deprecation_date: 1999-01-01
```

Comment:
I might get out-voted on this one, but I would love it if the majority of our examples are granular instants in time like this:
```suggestion
    deprecation_date: 1999-01-01 21:59:43.10-05:00
```

Something like this might be a little easier on the eyes while still being an instant in time:
```suggestion
    deprecation_date: 1999-01-01 00:00:00.00+00:00
```